Sofa/framework/Core/src/sofa/core/visual/VisualModel.h

```diff
@@ -48,9 +48,25 @@ class SOFA_CORE_API VisualModel : public virtual objectmodel::BaseObject
 public:
     SOFA_ABSTRACT_CLASS(VisualModel, objectmodel::BaseObject);
     SOFA_BASE_CAST_IMPLEMENTATION(VisualModel)
+
+    Data<bool> d_draw; ///< Display the visual model or not
+
+    /**
+     *  \brief Display the VisualModel object.
+     *
+     *  TODO(dmarchal, 2023-06-09): Deprecate VI et use NVI design pattern: In one year, remove the virtual keyword so that everyone
```

Comment:
```suggestion
     *  TODO(dmarchal, 2023-06-09): Deprecate VI and use NVI design pattern: In one year, remove the virtual keyword so that everyone
```